play-scala-scheduler-example/conf/routes

```diff
@@ -0,0 +1,12 @@
+# Routes
+# This file defines all application routes (Higher priority routes first)
+# https://www.playframework.com/documentation/latest/ScalaRouting
+# ~~~~
+
+# An example controller showing a sample home page
+GET     /                                   controllers.HomeController.index
+
+
+
+# Map static resources from the /public folder to the /assets URL path
+GET     /assets/*file               controllers.Assets.versioned(path="/public", file: Asset)

```

Comment:
There's no need for assets, the only route should even return a simple string "ready".

Reply:
Then, all files under `public/` and `conf/messages`  may be removed.